src/app/sales/sales.component.ts

```diff
@@ -11,27 +12,30 @@ import { SALESITEMS } from '../mock-data';
 })
 export class SalesComponent implements OnInit {
 
-  isLoadingItems = false;
 
   salesItems: SalesListItem[] = [];
-  constructor() {
+  constructor(private spinner: Ng4LoadingSpinnerService) {
   }
 
   ngOnInit() {
-    this.getSalesList();
+    //show spinner
```

Comment:
@ikramabumazen  why you returned them here :-1:  we approved them when they where inside getSalesList
:(

Reply:
@ikramabumazen  please fix this ASAP (as soon as possible )

Reply:
i fixed it :)